### DISCUSSION TITLE
Either set too look for the highest quality by default, or try to rewrite ani-cli for Animixplay for better library and quality.

### DISCUSSION THREAD

**Discussion by ghost** [pystardust/ani-cli#310] [General]
Title states it all.

**Comment by ghost** [comment]
If it already is set to find highest quality then there must be a new update that broke that setting. I've tried uninstalling and reinstalling both the AUR version plus directly from github, neither tries to find the highest quality by default.

**Comment by port19x** [comment]
Scraping gogo directly broke 3 days ago, resulting in very slow streaming speeds and only decent download speeds when using aria2c.
Right now we follow gogos download link over to sbplay and scrape from there.
But that approach isn't without issues either
